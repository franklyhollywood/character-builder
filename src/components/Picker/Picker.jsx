import React from 'react';

export default function Picker() {
	return <div>
        <label>Top
            <select value={top} onChange{(e) => onTopChange(e.target.value)}>
            {topPics.map((pic) => (
                <option key={pic}>{pic}</option>

            ))}
            </select>
        </label>

    </div>;
}
