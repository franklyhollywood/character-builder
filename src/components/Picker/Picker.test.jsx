import { screen, render } from '@testing-library/react';
import Picker from './Picker';

it('should render the dropdown components', () => {
	const { container } = render(<Picker />);
	const selectTop = screen.getByLabelText(/Top/);
	const selectMiddle = screen.getByLabelText(/Middle/);
	const selectBottom = screen.getByLabelText(/Bottom/);

	expect(selectTop).toBeInTheDocument();
	expect(selectMiddle).toBeInTheDocument();
	expect(selectBottom).toBeInTheDocument();
	expect(container).toMatchSnapshot();
});
