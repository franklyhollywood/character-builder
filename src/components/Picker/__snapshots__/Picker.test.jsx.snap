// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the dropdown components 1`] = `
<div>
  <div>
    <label>
      Top
      <select>
        <option>
          turkey
        </option>
        <option>
          fish
        </option>
        <option>
          monkey
        </option>
      </select>
    </label>
    <label>
      Middle
      <select>
        <option>
          torso
        </option>
        <option>
          tshirt
        </option>
        <option>
          tux
        </option>
      </select>
    </label>
    <label>
      Bottom
      <select>
        <option>
          legs1
        </option>
        <option>
          legs2
        </option>
        <option>
          cargo
        </option>
      </select>
    </label>
  </div>
</div>
`;
